jerry-core/include/jerryscript-core.h

```diff
@@ -346,6 +346,7 @@ bool jerry_value_is_object (const jerry_value_t value);
 bool jerry_value_is_promise (const jerry_value_t value);
 bool jerry_value_is_string (const jerry_value_t value);
 bool jerry_value_is_undefined (const jerry_value_t value);
+bool jerry_value_is_error (const jerry_value_t value);
```

Comment:
It would be better to keep these in sort. Please move it before ´jerry_value_is_function´. Or am I missing something?